osmosis/assetlist.json

```diff
@@ -16206,6 +16206,27 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/BRNZ.svg"
         }
       ]
-    }
+    },
+    {
+      "description": "Bernese Mountain Dog. Thematic dog token to have fun and be friends with all other dog tokens. Fixed supply 132M",
+      "denom_units": [
+        {
+          "denom": "factory/osmo1s6ht8qrm8x0eg8xag5x3ckx9mse9g4se248yss/BERNESE",
+          "exponent": 0
+        },
+        {
+          "denom": "BERNESE",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "sdk.coin",
+      "address": "osmo1s6ht8qrm8x0eg8xag5x3ckx9mse9g4se248yss",
+      "base": "factory/osmo1s6ht8qrm8x0eg8xag5x3ckx9mse9g4se248yss/BERNESE",
+      "name": "BERNESE",
+      "display": "BERNESE",
+      "symbol": "BERNESE",
+      "logo_URIs": {
```

Comment:
please define the `images`, which is a required property (logo_URIs) is the legacy property.

Reply:
done, sorry, I did not see it